services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -247,64 +256,22 @@ async def _update_impl(
         # single-update
         query = query.where(folders_v2.c.folder_id == folders_id_or_ids)
 
-    async with get_database_engine(app).acquire() as conn:
+    async with transaction_context(get_asyncpg_engine(app), connection) as conn:
         result = await conn.execute(query)
-        row = await result.first()
+        row = result.first()
```

Comment:
And this one. I would expect this one to be await maybe the `execute` above should be a `stream`?